src/components/album/AlbumPreview.js

```diff
@@ -31,6 +32,8 @@ const AlbumPreview = ({
   }, [apiUrl]);
 
   return (
+    <div>
+      <AlbumChat />
```

Comment:
Your `AlbumChat.js` component is currently expecting to be passed an `apiUrl` but here we're not passing it any props, so currently when that component loads the value will be undefined. If we want to pass the prop in we need to define it like an HTML attribute:

```suggestion
      <AlbumChat apiUrl={apiUrl} />
```

Reply:
And if you're looking for a way to test this locally, you can try creating a temporary route that loads this `AlbumChat` component with a static `apiUrl`

eg in App.js:

```js
import AlbumChat from './components/album/AlbumChat';

<Routes>
 ...all the existing routes
  <Route path="/chat" element={<AlbumChat apiUrl="https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy" />} />
</Routes>
```

(I just picked a random album from the db)

then you can navigate to `localhost:3000/chat` to view your component.

--- 

One thing to keep in mind, is that the album id in the Spotify URL is _Spotify's_ album ID; it won't match the Mongo Object Id that we identify the album by in our db. This may not matter if we're not using this id for anything other than naming the chat channel but just wanted to point it out in case you weren't aware.

Reply:
Thank you!!